packages/common/src/ceramic-api.ts

```diff
@@ -7,6 +7,21 @@ import type { GenesisCommit } from './index.js'
 import type { IndexApi } from './index-api.js'
 import { NodeStatusResponse } from './node-status-interface.js'
 
+/**
+ * Field definition for index
+ */
+export interface Field {
+  path: Array<string>
+}
+
+/**
+ * Index for a model
+ */
+export type ModelIndex = {
+  name: string
```

Comment:
I feel like the `name` field should be optional, and we should let the underlying database apply whatever default it wants if the user doesn't explicitly set a name